FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -451,7 +453,8 @@ - (void)fetchLatestConfig:(NSInteger)remainingAttempts targetVersion:(NSInteger)
                                         targetVersion:targetVersion];
                               }
                             }
-                          }];
+                          }
+               excludeEtagHeaderForRealtime:excludeEtagHeaderForRealtime];
```

Comment:
nit: can we update the method signature to have `excludeEtagHeaderForRealtime:` before `completionHandler:`?

Reply:
Think it would make more sense to just add it at the end for the method overall so that current calls to this method don't change too much.